src/testdrive/src/action.rs

```diff
@@ -75,15 +78,17 @@ pub struct State {
     aws_credentials: AwsCredentials,
     kinesis_client: KinesisClient,
     kinesis_stream_names: Vec<String>,
+    s3_client: S3Client,
+    s3_buckets_created: BTreeSet<String>,
 }
 
 impl State {
-    pub async fn reset_materialized(&mut self) -> Result<(), Error> {
+    pub(crate) async fn reset_materialized(&mut self) -> Result<(), Error> {
```

Comment:
I'd like to avoid changing things to `pub(crate)` arbitrarily. Testdrive is careful to only use `pub(crate)` where doing so actually restricts the visibility of an item. I think it's fine if other crates want to set different standards on that front, but I find confusing, and doubly so when there's no consistent standard within one crate.